openapi.json

```diff
@@ -1519,6 +1527,16 @@
                             "type": "string",
                             "description": "The time of last analysis for this cluster."
                           },
+                          "meta": {
+                            "type": "object",
+                            "properties": {
+                              "cluster_version": {
+                                "type": "string",
+                                "description": "The version of the cluster.",
```

Comment:
is it optional?

Reply:
No, the response will contain always the meta, even if it is empty. ("cluster_version": "")